go/service/user.go

```diff
@@ -344,6 +345,248 @@ func (h *UserHandler) UploadUserAvatar(ctx context.Context, arg keybase1.UploadU
 	return avatars.UploadImage(mctx, arg.Filename, nil /* teamname */, arg.Crop)
 }
 
+func (h *UserHandler) ProfileProofSuggestions(ctx context.Context, sessionID int) (ret keybase1.ProfileProofSuggestionsRes, err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G()).WithLogTag("US")
+	defer mctx.CTraceTimed("ProfileProofSuggestions", func() error { return err })()
+	suggestions, err := h.profileProofSuggestionsHelper(mctx)
+	if err != nil {
+		return ret, err
+	}
+	foldPriority := mctx.G().GetProofServices().SuggestionFoldPriority()
+	for _, suggestion := range suggestions {
+		if foldPriority > 0 && suggestion.Priority >= foldPriority {
+			ret.ShowMore = true
+			continue
+		}
+		ret.Suggestions = append(ret.Suggestions, keybase1.ProfileProofSuggestion{
+			Key:  suggestion.Key,
+			Text: suggestion.ProfileText,
+			Icon: suggestion.ProfileIcon,
+		})
+	}
+	return ret, nil
+}
+
+func (h *UserHandler) ProofSuggestions(ctx context.Context, sessionID int) (ret []keybase1.ProofSuggestion, err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G()).WithLogTag("US")
+	defer mctx.CTraceTimed("ProofSuggestions", func() error { return err })()
+	suggestions, err := h.profileProofSuggestionsHelper(mctx)
+	if err != nil {
+		return ret, err
+	}
+	for _, suggestion := range suggestions {
+		ret = append(ret, keybase1.ProofSuggestion{
+			Key:     suggestion.Key,
+			Text:    suggestion.PickerText,
+			Subtext: suggestion.PickerSubtext,
+			Icon:    suggestion.PickerIcon,
+		})
+	}
+	return ret, nil
+}
+
+type ProofSuggestion struct {
```

Comment:
can `keybase1.ProofSuggestion` also have these fields so there is just one type?